genesis-operation/check-vesting-validator-accs.ts

```
@@ -0,0 +1,163 @@
+
+const path = require("path");
+const { parse } = require("csv-parse/sync");
+
+function arrangeTokenAllocationInfoCsv(fileName: string) {
+  // read the file
+  const filepath = path.join(__dirname, fileName);
+  const data = fs.readFileSync(filepath);
+
+  // parse the content of the file
+  const records = parse(data);
+
+  // vestingKeyData = [address, balance, end_time]
+  let vestingKeyData = [];
+
+  // validatorVestingKeyData = [address, balance, end_time, initial_delegation_amount]
+  let validatorVestingKeyData = [];
+
+  for (const record of records) {
+    if (record[4] == "") {
+      continue;
+    } else if (record[8] == "TRUE" && record[11] == "") {
+      const balance = record[5].replace(/,/g, "");
+      const start_time = dateToUnixTime(record[9]);
+      const end_time = dateToUnixTime(record[10]);
+      const content = [record[4], balance, start_time, end_time];
+      vestingKeyData.push(content);
+    } else if (record[8] == "TRUE" && record[11] != "") {
+      const balance = record[5].replace(/,/g, "");
+      const start_time = dateToUnixTime(record[9]);
+      const end_time = dateToUnixTime(record[10]);
+      const initialDelegationAmount = record[11].replace(/,/g, "");
+      const content = [record[4], balance,start_time, end_time, initialDelegationAmount];
+      validatorVestingKeyData.push(content);
+    }
+  }
+  return { vestingKeyData, validatorVestingKeyData };
+}
+
+function checkVestingAccounts(vestingKeyData: any) {
+  for (const keyData of vestingKeyData) {
+    console.log("\n----------------------------");
+    const command = "ununifid q account " + keyData[0] + " -o json";
+    const vestingAccInfo = execCommand(command);
+    const jsonAcc = vestingAccInfo.replace("[", "").replace("]", "");
+    const json = JSON.parse(jsonAcc);
+
+    // check original vesting amount
+    if (keyData[1] !== json.base_vesting_account.original_vesting.amount) {
+      console.error("------Below information has wrong value------");
```

Should I just display this in the console?
For example, you can throw an error and stop the process.